src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+
```

Comment:
create a constructor with String color argument and call `super(color)` where you call `setColor` in other classes, it's the more traditional approach